av/container/core.pyx

```diff
@@ -203,6 +229,11 @@ cdef class Container(object):
         if sentinel is not _cinit_sentinel:
             raise RuntimeError('cannot construct base Container')
 
+        timeouts = options.pop('timeouts', {}) if options else {}
+        timeouts = timeouts if timeouts is not None else {}
+        self.open_timeout = int(timeouts.get('open_timeout', 30000))
```

Comment:
I'm not sure I'm comfortable with using timeouts by default. @mikeboers any thoughts?